ts/lib/auth/computeclient.ts

```diff
@@ -39,9 +39,11 @@ export class Compute extends OAuth2Client {
    *
    * Retrieve access token from the metadata server.
    * See: https://developers.google.com/compute/docs/authentication
+   *
+   * @param {number=} refreshTokeEarlyMillis The token should be refreshed if it will expire within this many milliseconds.
```

Comment:
Missing the `n` in `refreshTokenEarlyMillis`

Reply:
Fixed.